### PR TITLE
Update to version 0.3.3 in README.md and mkdocs.yml

### PR DESCRIPTION
**Pull request by miguelpoyatosmora**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -47,5 +47,5 @@ nav:
 extra:
   fleet:
     apiVersion: fleet.cattle.io/v1alpha1
-    version: v0.3.0
+    version: v0.3.3
     helmversion: 0.3.0
```
Comment (nickgerace) [inline mkdocs.yml:51]: Wondering if we have to worry about `helmversion`. CC: @StrongMonkey

Comment (StrongMonkey) [reply mkdocs.yml:51]: yeah we should update this too. @miguelpoyatosmora Can you also update it

Comment (miguelpoyatosmora) [reply mkdocs.yml]: Oh, thanks! what about this?

Comment (StrongMonkey) [reply mkdocs.yml]: Thanks! LGTM now

**Review by StrongMonkey — APPROVED** [review]

**Pull request merged**